custom-vu/src/main/kotlin/jces1209/vu/page/IssueNavigator.kt

```diff
@@ -0,0 +1,9 @@
+package jces1209.vu.page
+
+import org.openqa.selenium.WebElement
+
+interface IssueNavigator {
+    fun waitForNavigator()
+    fun getIssueElementFromList(): WebElement
+    fun selectIssue(element: WebElement)
```

Comment:
What about to encapsulate work on selenium objects and merge methods to one?
`getIssueElementFromList()` + `selectIssue()` = `selectOtherIssue()`

Reply:
In WorkOnSearchResults action code firstly check that all is loaded and select some element with which going to work. After that in measure section works with that element

val element = issueNavigator.getIssueElementFromList()

        meter.measure(
            key = SWITCH_BETWEEN_ISSUES_IN_SEARCH_RESULTS,
            action = {
                issueNavigator.selectIssue(element)
            }
        )

Reply:
I don't suggest to merge `issueNavigator.waitForNavigator()`
Selecting issue could be a part of the switching issue - it should be no problem to measure it
so the client of page object doesn't work with WebElement objects